xml/System.Printing/PrintQueueCollection.xml

```diff
@@ -338,9 +338,25 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns an object that implements the generic <see cref="T:System.Collections.IEnumerator" /> interface that has been closed with <see cref="T:System.Printing.PrintQueue" />.</summary>
+        <returns>An object that implements the generic <see cref="T:System.Collections.IEnumerator" /> and that can iterate through the <see cref="T:System.Printing.PrintQueue" /> objects that the <see cref="T:System.Printing.PrintQueueCollection" /> contains.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+We recommend that you use the `foreach` statement to iterate through all members of the collection. Use <xref:System.Printing.PrintQueueCollection.GetEnumerator%2A> only when you need to customize iteration, such as by skipping every other member.
+
+## Examples
```

Comment:
The example should be removed -- it calls GetEnumerator, not IEnumerable.GetEnumerator.